### ISSUE TITLE
A11y_mssql for VSCode_Select top 1000 _Keyboard:When using arrow keys to move to "split editor right" icon, tooltip is not available with the keyboard keys.

### ISSUE THREAD

**Issue by Roopesh-T** [microsoft/vscode-mssql#17202]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS;#A11yTCS;#BM_mssql for VSCode_Jan2022;#mssql for VSCode;#Win32;#FTP;#DesktopApp;#A11ySev1;#Keyboard;#AINotInScope;#WCAG2.1.1;#Regressed:02-03-2023;

## Environment Details:
Application Name: mssql for VSCode
Windows Version: Win11

## Repro Steps:

1. Open Visual studio code 
2. Tab till extension button and search for "mssql"
3. Tab till install and move to the SQL Server  button
4. Tab till Add Connection Button and insert the valid credentials
5. After adding the server move to database expand it -->System database expand-->Master expand-->Tables expand -->System table expand and right click on it and select top 1000 option
6. Select top 1000 window will open
7. Tab to the "split editor right" icon and check if the tooltip is available for the icon.

## Actual:
When using arrow keys to move to "split editor right" icon, tooltip is not available with the keyboard keys.

#### Similar issue is also observed in the following screens

1. Select top 1000 _Execute icon tooltip having shortcut key in it.
2. Select top 1000 Results window_Icons Beside table in the results "Save as CSV, Save as JSON, Save as excel"

## Expected:
When using arrow keys to move to "split editor right" icon, tooltip should available with the keyboard keys which is available with mouse, Since shortcut keys were also mentioned in the tooltip.

## User Impact:
Keyboard users will not get the tooltip information.

## Attachment
![17202](https://user-images.githubusercontent.com/86601127/149272498-8ca864ba-ffe8-4ccf-9a2d-a8af98f3e649.gif)

**Comment by smartguest** [comment]
Closing as duplicate of #17205 (Same base vscode issue)

**Comment by Roopesh-T** [comment]
This bug can be closed by trusted testers by C + AI team after bug verification. Can you please change the bug to open state.
The duplicate bug mentioned is not the same bug as this, if it is a duplicate, please provide the exact same bug link.

**Comment by Roopesh-T** [comment]
@smartguest The duplicate bug you mentioned got closed, but this issue is still repro in the latest env 
![image](https://user-images.githubusercontent.com/86601127/216575569-acdc8f4a-0d0c-4911-8672-31a37b6a4167.png)
please reopen the bug.
![17202_Repro](https://user-images.githubusercontent.com/86601127/216589760-9d4edb26-ede5-490f-8380-e37500863de5.gif)

**Comment by alanrenmsft** [comment]
@Roopesh-T please close as a dupe of https://github.com/microsoft/vscode/issues/52016, see comments here: https://github.com/microsoft/vscode-mssql/issues/1231#issuecomment-521351427.

**Comment by smartguest** [comment]
This is a dupe of https://github.com/microsoft/vscode/issues/132344 as well.

**Comment by Roopesh-T** [comment]
As the above mentioned bug https://github.com/microsoft/vscode/issues/132344 is not exact same control, but the root cause is same, we cannot close this bug, but instead we have to follow the [external bug process](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDEzODMxNDJkZjM4NzQwYWZhMjJiOWY0MWJkMjEwYmNjfDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHwzMTU1Mzc4OTc1OTk5OTk5OTk5fEdvb2R8VkdWaGJYTlRaV04xY21sMGVWTmxjblpwWTJWOGV5SldJam9pTUM0d0xqQXdNREFpTENKUUlqb2lJaXdpUVU0aU9pSWlMQ0pYVkNJNk1USjk%3D&sdata=RXhWOTl1aDRoSlhqK3F2MkxKamRKUEhVNm1PZmxPSVRncUR5Y0VENkV1Zz0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2Cv-bbarik%40microsoft.com&OR=Teams-HL&CT=1625140596391).

**Comment by alanrenmsft** [comment]
@Roopesh-T It is actually the same thing, those are not created by mssql extension, those are vscode UI elements, once they fix it, we will get it automatically, definitely not an external bug.

**Comment by fsteffi** [comment]
GithubTags:#LinkedBug:132344;#External;#External:VSCode;#LinkedBug;

https://github.com/microsoft/vscode/issues/132344

**Comment by Roopesh-T** [comment]
Lowering the bug severity to sev3

**Comment by keerthiduvvuri** [comment]
GithubTags:#ResolvedByTT

**Comment by laurenastrid1** [comment]
Please use the latest version of the VS Code extension to verify and close this bug. This bug is no longer present in the extension.

**Comment by laurenastrid1** [comment]
Closing because UI no longer exists.
